src/Razor/src/Microsoft.VisualStudio.Editor.Razor/DefaultRazorDocumentManager.cs

```diff
@@ -20,31 +20,26 @@ internal class DefaultRazorDocumentManager : RazorDocumentManager
 
         [ImportingConstructor]
         public DefaultRazorDocumentManager(
-            ForegroundDispatcher dispatcher,
+            ForegroundDispatcher foregroundDispatcher,
```

Comment:
Pretty sure this doesn't need the dispatcher anymore. I'd imagine this would expect everything to be UI Thread related. One of the key reasons is that a lot of the older stack didn't know about the ProjectSNapshotManager at all but utilized the ForegroundDispatcher as a mechanism to assert "is ui thread" which was a bit backwards 😄. We made big mistakes in the past